### DISCUSSION TITLE
Change "review requested" message

### DISCUSSION THREAD

**Discussion by dolfinus** [community/community#113255] [Mobile]
I've subscribed to notifications on this PR:
https://github.com/python/cpython/pull/116392

After PR author requested a review from repo maintainers, I've got notification with message "@user requested *your* review".
<details>
<summary>Screenshot</summary>

![Screenshot_20240319-105015_GitHub.png](https://github.com/community/community/assets/4661021/513e8dd8-7c1f-4844-b647-af1b40134397)
</details>

But I'm not a repo maintainer, and review was requested for another user. Can you please change the message to "@user requested *a* review"?

**Comment by esahmed21** [comment]
Thank you @dolfinus for bringing this to our attention! I'll promptly create an internal ticket for our team to investigate and address this matter.
